drivers/adc/ad7768-1/ad77681.c

```diff
@@ -117,6 +117,7 @@ int32_t ad77681_spi_reg_read(struct ad77681_dev *dev,
 	uint8_t buf[3], crc_buf[2];
 	uint8_t buf_len = (dev->crc_sel == AD77681_NO_CRC) ? 2 : 3;
 
+	reg_data[0] = AD77681_REG_READ(reg_addr);
```

Comment:
could you describe the commit a bit more. I think the buffer contains register data, or that is what the code is telling me.